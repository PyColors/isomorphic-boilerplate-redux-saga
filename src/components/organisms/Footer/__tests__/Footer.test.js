import renderer from 'react-test-renderer';
import React from 'react';
import footer from '../Footer';

describe('The Footer <Component', () => {
  describe('The display element', () => {
    it('Should not regress', () => {
      const tree = renderer.create(<footer />);

      expect(tree.toJSON()).toMatchSnapshot();
    });
  });
});
