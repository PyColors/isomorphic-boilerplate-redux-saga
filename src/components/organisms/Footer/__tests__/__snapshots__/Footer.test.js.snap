// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer <Component The display element Should not regress 1`] = `<footer />`;
